docs/api/app.md

```diff
@@ -228,6 +228,10 @@ app.on('login', function(event, webContents, request, authInfo, callback) {
 
 Emitted when the gpu process crashes.
 
+### Event: 'dark-mode-changed' _OS X_
```

Comment:
I'm :+1: on that, it'd also be great if someday (or maybe even in this PR!) we could detect GTK+ and Windows OS theme changes